pkg/virt-launcher/virtwrap/network/podinterface_test.go

```diff
@@ -294,6 +295,7 @@ var _ = Describe("Pod Network", func() {
 			mockNetwork.EXPECT().GetMacDetails(podInterface).Return(fakeMac, nil)
 			mockNetwork.EXPECT().LinkSetMaster(dummy, bridgeTest).Return(nil)
 			mockNetwork.EXPECT().AddrDel(dummy, &fakeAddr).Return(errors.New("device is busy"))
+			mockNetwork.EXPECT().IsIpv4Primary().Return(true).AnyTimes()
```

Comment:
will first make sure, it affect only this test ,else it need to be specific times